Puppetfile

```diff
@@ -4,6 +4,6 @@
 # The following directive installs modules to the managed moduledir.
 moduledir '.modules'
 
-mod 'WhatsARanjit/node_manager', '0.7.5'
+mod 'node_manager', '1.0.1'
```

Comment:
```suggestion
mod 'puppetlabs/node_manager', '1.0.1'
```

Reply:
Why is that Puppetfile even in this module?

Reply:
I believe the intention was to improve instructions around installation and usage. Will keep it for now, and move or remove when this is done.

Reply:
But where is that file used or documented? The installation instructions in `/documentation/` use `bolt project init .. --modules puppetlabs-peam`. That command doesn't use the Puppetfile, only metadata.json.

Reply:
@bastelfreak I missed this comment, the documentation side hadn't been done as of yet. I'll raise a ticket to sort it.